src/test/java/ca/corefacility/bioinformatics/irida/service/impl/integration/ProjectServiceImplIT.java

```diff
@@ -656,6 +657,7 @@ public void testGetPagedProjectRevisions() {
 				mostRecent.getEntity().getName());
 	}
 
+	@Ignore
```

Comment:
Since spring-data-envers 2.1.X you can now retrieve revisions for deleted entities. So these tests could be removed or refactored, also the code that throws EntityRevisionDeletedException could be removed. Let me know how you would like to proceed.